src/client.js

```diff
@@ -189,12 +189,14 @@ export default class Client {
 
     return Promise.all([baseQueryPromise, ...disjunctiveQueriesPromises]).then(
       ([baseQueryResults, ...disjunctiveQueries]) => {
-        disjunctiveQueries.forEach(disjunctiveQueryResults => {
-          const [facetName, facetValue] = Object.entries(
-            disjunctiveQueryResults.facets
-          )[0];
-          baseQueryResults.facets[facetName] = facetValue;
-        });
+        if (baseQueryResults.facets) {
```

Comment:
I believe its not enough to check whether facets is present, you will need to check the number of keys within the object too.

Reply:
Hi Joel, good observation I just added the extra check